packages/swamp/src/connectors/jsonplaceholder.ts

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+import { z } from 'zod';
+import { Inserts, Loader, LoaderResponse } from '../core/types';
+
+export const Cursor = z.object({
+  currentObject: z.enum(['photos', 'albums']),
+});
+export type Cursor = z.infer<typeof Cursor>;
+
+type Photos = {
+  albumId: number;
+  id: number;
+  title: string;
+  url: string;
+};
+type Albums = {
+  userId: number;
+  id: number;
+  title: string;
+};
+
+const getPhotos = async (): Promise<Photos[]> => {
+  try {
+    const response = await axios.get('https://jsonplaceholder.typicode.com/photos');
+    return response.data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getAlbums = async (): Promise<Albums[]> => {
+  try {
+    const response = await axios.get('https://jsonplaceholder.typicode.com/albums');
+    return response.data;
+  } catch (error) {
+    throw error;
+  }
```

Comment:
What's the purpose of catching and rethrowing the error? Lots of theses try catch blocks

Reply:
Many such catches